modules/cco-ccoctl-configuring.adoc

```diff
@@ -37,6 +37,45 @@ endif::alibabacloud[]
 The `ccoctl` is a Linux binary that must run in a Linux environment.
 ====
 
+.Prerequisites
+
+ifdef::aws-sts[]
+* You have created an AWS account for the `ccoctl` to use with the following permissions:
+
+.Required AWS permissions
+[cols="a,a"]
+|====
+|`iam` permissions |`s3` permissions
+
+|* `iam:CreateOpenIDConnectProvider`
+* `iam:CreateRole`
+* `iam:DeleteOpenIDConnectProvider`
+* `iam:DeleteRole`
+* `iam:DeleteRolePolicy`
+* `iam:GetOpenIDConnectProvider`
+* `iam:GetRole`
+* `iam:GetUser`
+* `iam:ListOpenIDConnectProviders`
+* `iam:ListRolePolicies`
+* `iam:ListRoles`
+* `iam:TagOpenIDConnectProvider`
+|* `s3:CreateBucket`
+* `s3:DeleteBucket`
+* `s3:DeleteObject`
+* `s3:GetBucketAcl`
+* `s3:GetBucketTagging`
+* `s3:GetObject`
+* `s3:GetObjectAcl`
+* `s3:GetObjectTagging`
+* `s3:PutBucketAcl`
+* `s3:PutBucketTagging`
+* `s3:PutObject`
+* `s3:PutObjectAcl`
+* `s3:PutObjectTagging`
```

Comment:
here is the list of the permissions that are required for ccoctl. I have tested it and works fine for me. There were some missing permissions in there which I have added now.

```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "VisualEditor0",
            "Effect": "Allow",
            "Action": [
                "iam:GetRole",
                "s3:GetBucketTagging",
                "s3:CreateBucket",
                "s3:ListBucket",
                "iam:CreateRole",
                "iam:TagRole",
                "iam:PutRolePolicy",
                "iam:ListRoles",
                "iam:DeleteRole",
                "s3:GetBucketAcl",
                "iam:CreateOpenIDConnectProvider",
                "s3:PutObject",
                "s3:GetObjectAcl",
                "s3:GetObject",
                "s3:PutBucketTagging",
                "iam:ListOpenIDConnectProviders",
                "iam:DeleteRolePolicy",
                "s3:GetObjectTagging",
                "s3:PutBucketAcl",
                "s3:PutObjectTagging",
                "iam:GetUser",
                "s3:DeleteObject",
                "iam:ListRolePolicies",
                "iam:DeleteOpenIDConnectProvider",
                "iam:GetOpenIDConnectProvider",
                "s3:DeleteBucket",
                "iam:TagOpenIDConnectProvider",
                "s3:PutObjectAcl"
            ],
            "Resource": "*"
        }
    ]
}
```